tests/e2e/data/_expected_context.yaml

```diff
@@ -0,0 +1,48 @@
+segmentItems:
```

Comment:
`tests/e2e/data/_expected_context.yaml` This file seems a temp file?

Reply:
Yes. deleted.